pkg/middleware/types.go

```diff
@@ -0,0 +1,7 @@
+package middleware
+
+type AccessProfileRequest struct {
+	RedirectURL    string `json:"redirect_url"`
```

Comment:
Why do you need to take this from the body? Don't we get it from the table?

Reply:
we need to verify the redirect url also, because an app can have multiple redirect url and we need to verify that is it among the listed one's or not

Reply:
Ok